proj/src/main/webapp/chat.js

```diff
@@ -318,18 +318,33 @@ const LIMIT = 20; // how many messages to load at a time
 function initChat() {
     firebase.auth().onAuthStateChanged(async firebaseUser => {
         if(firebaseUser){
-            setupSidebar();
-            clickWithEnterKey();
-            notifications.initNotifications();
-
-            // InitUserChat sets information relevant to logged-in user
-            // Must run before enclosed functions
-            await initUserChat().then(function(){
-                populateSidebar();
-                initRef();
-                populateProfileSidebar(firebaseUser.uid);
-                addBlockedToSettings();
-            });
+            // If the user is verified or using a test server
+            if(firebaseUser.emailVerified || window.location.href.includes("https://8080")){
```

Comment:
It seems weird to have something for the devs in the deployed code. Is there any other way to do this? Like maybe we can check if the user has a specific exemption property in the database that the devs can manually add from the firebase database page?

Reply:
Ideally we would be able to create admin accounts and whatnot, but given the very constrained time at this point I don't think it makes sense to build an admin/dev account feature.